sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedResponseBase.java

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.implementation.http;
+package com.azure.core.http.rest;
```

Comment:
Some of the classes in implementation packages were lean on javadocs (like this one is missing javadoc for both constructors).  Maybe it's hard to catch all of them here. We should add checkstyle rule to report missing javadocs  on public methods/classes in non-impl packages.

Reply:
I've added additional checkstyle rules and will add more javadocs as it identifies issues.